src/librbd/mirror/snapshot/UnlinkPeerRequest.cc

```diff
@@ -120,6 +135,10 @@ void UnlinkPeerRequest<I>::handle_unlink_peer(int r) {
   ldout(cct, 15) << "r=" << r << dendl;
 
   if (r == -ERESTART || r == -ENOENT) {
+    if (r == -ERESTART) {
```

Comment:
Need to add a unit test to `src/test/librbd/mirror/snapshot/test_mock_UnlinkPeerRequest.cc` for this.

Reply:
I added `UnlinkErrorRestart` to cover this.